utils/Command/MissingInSetCommand.php

```diff
@@ -23,70 +23,73 @@
 final class MissingInSetCommand extends Command
 {
     /**
-     * @var array<string, string>
+     * @var list<array<string, string>>
      */
     private const RULES_DIRECTORY_TO_SET_CONFIG = [
-        __DIR__ . '/../../rules/CodeQuality/Rector' => __DIR__ . '/../../config/set/code-quality.php',
```

Comment:
Hm, this might make the whole category = set overly complex :) I want to avoid it and keep it 1:1